pom.xml

```diff
@@ -29,6 +29,8 @@
         <commons-io.version>2.7</commons-io.version>
         <messaging.saaj.version>1.5.2</messaging.saaj.version>
         <cloudinary.version>1.21.0</cloudinary.version>
+        <aws-java-sdk.version>1.11.155</aws-java-sdk.version>
+        <javers.version>6.14.0</javers.version>
```

Comment:
Could be 7.x?

Reply:
We need to migrate to javax.persistence to jakarta to do this